app.js

```diff
@@ -262,6 +266,12 @@ function tick() {
   if (liveCells === 0) {
     const stopButton = document.querySelector('#stop');
     stopButton.click();
+
```

Comment:
To me it seems a little strange to call a button click from w/in the JS. The alternative would be calling the functions that 'click' calls.

Having said that, one problem you'd have with that is passing a reference of the timerObject to clearInterval (in next comment). RN with how you're using localStorage your current clicking button internally seems like the best option (at least I couldn't think of any other soln)